source/overloading.tex

```diff
@@ -1698,13 +1717,76 @@
 any argument for which there is no corresponding parameter is
 considered to ``match the ellipsis''\iref{over.ics.ellipsis}.
 \item
-A candidate function having more than $m$ parameters is viable
-only if all parameters following the $m^\text{th}$
-have default arguments\iref{dcl.fct.default}.
+A candidate function \tcode{C} having more than $m$ parameters is viable
+only if the set of scopes $G$, as defined below, is not empty.
+$G$ consists of every scope $X$ that satisfies all of the following:
+\begin{itemize}
+\item There is a declaration of \tcode{C}, whose host scope is $X$,
+considered by the overload resolution.
+\item For every $k^\textrm{th}$ parameter $P$ where $k$ > $m$,
+there is a reachable declaration, whose host scope is $X$,
+that specifies a default argument\iref{dcl.fct.default} for $P$.
+\end{itemize}
+If \tcode{C} is selected as the best viable function\iref{over.match.best}:
+\begin{itemize}
+\item
+$G$ shall contain exactly one scope (call it $S$).
+\item
+If the candidates are denoted by a \grammarterm{splice-expression},
+then $S$ shall not be a block scope.
+\item
+The default arguments used in the call \tcode{C} are
```

Comment:
As discussed offline, probably should have been "the call ___to___ `C`".

Reply:
@jensmaurer Is that OK as an editorial follow-up? It's not in the paper.